pyproject.toml

```diff
@@ -85,6 +85,8 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 datasets = [
+    # faster-coco-eval 1.6.4+ required for numpy>=2 support
+    "faster-coco-eval>=1.6.4",
```

Comment:
For more stable operation, I would suggest using 1.6.6

faster-coco-eval>=1.6.6